test/conftest.py

```diff
@@ -85,17 +85,6 @@ def pytest_addoption(parser):
     )
 
 
-def pytest_configure(config):
-    """Make sure env vars are set before unit tests.
-
-    io_managers unit tests need these to be set, but they don't have to point to
-    anything meaningful. They will be reset by the `pudl_env` fixture before being used.
-    """
-    os.environ["PUDL_OUTPUT"] = os.getenv("PUDL_OUTPUT", "~/")
-    os.environ["DAGSTER_HOME"] = os.getenv("DAGSTER_HOME", "~/")
-    os.environ["PUDL_INPUT"] = os.getenv("PUDL_INPUT", "~/")
-
-
```

Comment:
@zschira this fixture was creating a `~/` directory in my pudl directory and I don't really understand its purpose.

Reply:
I originally added this because pytest will automatically run this fixture before anything else, and it allowed the unit tests to pass when they were ran without `PUDL_OUTPUT`/`PUDL_INPUT` being set. I just checked though, and it seems they're passing without this fixture now, so it should be fine to remove.

Reply:
Ok! Do you know why they aren't required anymore?